### ISSUE TITLE
[Suggestion] Early game one way EMC trashcan

### ISSUE THREAD

**Issue by Sandstroem** [sinkillerj/ProjectE#1767]
Hello,

I would like to make a suggestion for a new block. The early game EMC trashcan.

A lot of modpacks use ProjectE in a way, that it is gated behind complicated recipes, to make it more of an endgame mod. However, often right from the start, you get so much stuff that it clutters your storage system. That is where the EMC trashcan would come in handy.

Here is how it works: Imagine a Extra Utilities trashcan. You throw stuff in, it gets turned into EMC that gets stored in your EMC data. Since you don't have a table yet, you can not convert it into new items.
You could still give the trash can the feature that it learns the items, not sure about that, but it should not allow you to get items.

Packs I played with ProjectE were different variants of Project Ozone and now Awakening. Especially on Awakening you get so much stuff, that clutters dozens of chests, that a "Burn to EMC and store for later" mechanic would be really cool to have.

Have a nice day

Sandstroem

**Comment by preceptor-teeth** [comment]
This mod adds a way that would work with this idea:  https://minecraft.curseforge.com/projects/equivalent-integrations

**Comment by piisawheel** [comment]
What do you mean _early game?_ If you are focused enough, you can gab some seeds -> bread, stone pickaxe to the bottom of the map, veinmine 5x5 areas, get your iron/diamonds/redstone/obsidian. Goto the nether and grab some glowstone.  I often have a transmutation tablet in less than an hour... Am I doing it wrong?

**Comment by MaPePeR** [comment]
@piisawheel there are a lot of modpacks that alter the recipe in a way that this is not possible.
Gating it behind draconic evolution or something else that takes more effort.

**Comment by piisawheel** [comment]
@MaPePeR Fair Enough.... I guess in the mean time you'll have to develop a hoarding habit.
